File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ride/UpdateTripDataResponse.java
package com.lattis.ellipse.data.network.model.response.ride;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 7/28/17.
 */

public class UpdateTripDataResponse {

    @SerializedName("duration")
    private double duration;

    @SerializedName("charge_for_duration")
    private float charge_for_duration;

    @SerializedName("currency")
    private String currency;


    @SerializedName("ended_trip")
    UpdateTripEndedResponse updateTripEndedResponse;


    public double getDuration() {
        return duration;
    }

    public float getCharge_for_duration() {
        return charge_for_duration;
    }

    public String getCurrency() {
        return currency;
    }


    public UpdateTripEndedResponse getUpdateTripEndedResponse() {
        return updateTripEndedResponse;
    }




}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/ride/UpdateTripDataResponse.kt
package com.lattis.data.entity.response.ride

import com.google.gson.annotations.SerializedName

class UpdateTripDataResponse {

    @SerializedName("duration")
    val duration: Double = 0.toDouble()

    @SerializedName("charge_for_duration")
    val charge_for_duration: Float = 0.toFloat()

    @SerializedName("currency")
    val currency: String? = null

    @SerializedName("bike_battery_level")
    val bike_battery_level: String? = null


    @SerializedName("ended_trip")
    var updateTripEndedResponse: UpdateTripEndedResponse? = null
        internal set


}
